src/utils/stac.js

```diff
@@ -433,7 +433,11 @@ export const renderItemColumn = (item, _, column) => {
     case "shape":
     case "chunks":
       if (Array.isArray(fieldContent)) {
-        fieldContent = fieldContent.join(", ");
+        let newContent = fieldContent.map(v => {
+          return (Number.isNaN ? 'varies' : v);
```

Comment:
Style question @mmcfarland: do you prefer `Number.isNan` or `isNil` here? In the JSON, the values are `null`.

Reply:
I'm pretty sure `Number.IsNaN(null)` will be false, so `isNil` is probably a better bet here.

For a general style comment, I would probably chain the calls to something like: 

```js
fieldContent = fieldContent
    .map(v => isNil(v) ? 'varies' : v)
    .join(", ")
```